src/arguments.js

```diff
@@ -24,7 +24,7 @@ const parser = new argparse.ArgumentParser({
 });
 parser.addArgument(["-p", "--port"], {
     type: "int",
-    defaultValue: 8060,
+    defaultValue: process.env.PORT || 8060,
```

Comment:
Interesting that this default was 8060 but we were using 8080 above - is that ok?

Reply:
Yeah...I think I might change this default to 8080 just because. I can see no reason to keep 8060 and GAE is always going to give us a PORT anyway.